haiku-apps/yab_ide/yab_ide-2.2.7.recipe

```diff
@@ -6,12 +6,12 @@ much more.
 yab wouldn't be complete without the yab-IDE, a powerful development \
 environment, which of course is programmed in yab itself."
 HOMEPAGE="http://yab.orgfree.com"
```

Comment:
@bbjimmy, could you please add a trailing slash to HOMEPAGE?
(Sorry for my being picky, and thanks in advance ;-)